src/api/zoops.ts

```diff
@@ -118,4 +118,25 @@ zoopsRouter.post("/:id/faves", requireUser, async (req: any, res, next): Promise
     }
 })
 
+  // DELETE /api/zoops/:id/faves
+  zoopsRouter.delete("/:id/faves", requireUser, async (req: any, res, next): Promise<void> => {
+    const userId = req.user?.id;
+
+    try {
+        const { id } = req.params;
+        const fave = await prisma.fave.findFirstOrThrow({
+            where: {
+                zoopId: Number(id), 
+                faverId: Number(userId)
```

Comment:
I would not recommend sending the faverId as part of find. Instead, find the fave, and then check if the fave.faverID = req.user.id

If not, set status to 403 and send a forbidden error

Reply:
But can't zoops have more than one fave? Finding the first fave for a zoopId might miss the user's fave, right? Should I get an array of faves and check their faverIds against the user id?

Reply:
OH wait, I take it back, I made an error here. The route should not be `/zoops/:id/faves` in this case - it would be expected that would delete all faves on a zoop.

I would recommend picking one of these:
- `DELETE /zoops/:id/faves/:faveId`
- `DELETE /faves/:id`

Reply:
Refactored in PR #40